paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -469,10 +469,16 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
         } else {
             null
         }
+        val customerEmail = config?.customer?.let {
```

Comment:
I think we should use the email from config if it's provided, and fetch it as a second option (same on the FlowController).

```suggestion
        val customerEmail = config?.defaultBillingDetails?.email ?: config?.customer?.let {
```